README.md

```diff
@@ -87,6 +87,22 @@ mvn clean verify \
 -Dcromwell-host=http://cromwell-url:8000
 ```
 
+## Commands
+
+This section lists command(s) run by fastqc workflow
+
+* Running fastqc
+
+fastqc workflow runs the following command (excerpt from .wdl file). INPUT_FASTQ is a placeholder for an input file.
+
+```
+set -euo pipefail
```

Comment:
is it important to include this line if the purpose is to describe how the tool is run?

Reply:
Not really, we can remove this line